recipes/trinity/build.sh

```diff
@@ -20,5 +20,21 @@ cd $BINARY_HOME && ln -s $TRINITY_HOME/Trinity $BINARY
 ln -s $TRINITY_HOME/util/* .
 ln -s $TRINITY_HOME/Analysis/DifferentialExpression/PtR
 ln -s $TRINITY_HOME/Analysis/DifferentialExpression/run_DE_analysis.pl
+
+ln -s $TRINITY_HOME/trinity-plugins/collectl/bin/client.pl
+ln -s $TRINITY_HOME/trinity-plugins/collectl/bin/col2tlviz.pl
+ln -s $TRINITY_HOME/trinity-plugins/collectl/bin/collectl
+ln -s $TRINITY_HOME/trinity-plugins/collectl/bin/collectl2html
+ln -s $TRINITY_HOME/trinity-plugins/fastool/fastool
+ln -s $TRINITY_HOME/trinity-plugins/jellyfish/bin/jellyfish
+ln -s $TRINITY_HOME/trinity-plugins/parafly/bin/ParaFly
+ln -s $TRINITY_HOME/trinity-plugins/Trimmomatic/trimmomatic.jar
+ln -s $TRINITY_HOME/trinity-plugins/slclust/bin/slclust
+ln -s $TRINITY_HOME/trinity-plugins/BIN/samtools
```

Comment:
Many of these are already available as separate packages. It is not desirable to have trinity copy them over the binaries of separate packages. Can you please only add those which are trinity-only?